### PR TITLE
README: doc the exclude function

### PR DESCRIPTION
**Pull request by vbatts**
https://github.com/vbatts/git-validation/pull/21

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ vbatts@valse ~/src/vb/git-validation (master) $ echo $?
 1
 ```
 
+Excluding paths that are out of the scope of your project:
+```bash
```
Comment (wking) [inline README.md]: This isn't Bash, it's a snippet of an interactive Bash session.  If you want to highlight it, I recommend [`console`][1].

[1]: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4090

Comment (vbatts) [reply README.md]: woah

Comment (wking) [reply README.md]: Don't ask me [how I know this][1] ;).

[1]: https://github.com/wking/oci-command-line-api/commit/770453114b748d4746ba1614a1e9a5d9003803fe

Comment (vbatts) [reply README.md]: heh

**Comment by vbatts** [comment]
LGTM

**Pull request merged**